# this service wraps the transformQuaternion method in TransformListener (both versions).
# if fixed_frame is empty the non-interpolating version is called and time is ignored.

string target_frame
time   target_time 
robot_msgs/QuaternionStamped qin
string fixed_frame
---
robot_msgs/QuaternionStamped qout